test/Kestrel.Core.Tests/HttpConnectionTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'm not really sure. VS shows the encoding of this, and other files in the repo that I didn't touch, as "Western European Code Page 1252" which is the ANSI Latin-1 code page (aka ASCII, for all intents and purposes). I don't know what it was before. Perhaps because I edited it and saved it at some point it converted it?